src/assets/scss/app.scss

```diff
@@ -54,13 +54,14 @@
 
 // vendor styles
 @import "vendor/aos";
-@import "vendor/slick-carousel";
 @import "vendor/index-menu";
+@import "vendor/slick-carousel";
 
 // mixins
+@import 'mixins/avatar';
 @import 'mixins/container-border';
 @import 'mixins/flex';
-@import 'mixins/avatar';
+@import 'mixins/shrunk-sidebar';
```

Comment:
Why sorting in the alphabetical order ? Is there any reason for it ?

We generally try to sort simport from the more generic to the more specific and application-oriented, since the last styles imported will override the first ones.